qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -739,7 +739,14 @@ def on_load_finished(self, tab, ok):
             start = config.val.colors.tabs.indicator.start
             stop = config.val.colors.tabs.indicator.stop
             system = config.val.colors.tabs.indicator.system
-            color = utils.interpolate_color(start, stop, 100, system)
+            try:
+                qurl = self.widget.tab_url(idx)
+                if (qurl.scheme(), qurl.host()) == ('qute', 'back'):
+                    color = config.val.colors.tabs.indicator.suspended
+                else:
+                    color = utils.interpolate_color(start, stop, 100, system)
+            except:
```

Comment:
Like @toofar said in his earlier review, a bare `except:` is [never a good idea](https://docs.python.org/2/howto/doanddont.html#except). What exception happens here, and why?